<launch>
    <node pkg="planar_objects" type="box_detector" respawn="true" output="screen" name="box_detector">
      <remap from="stereo" to="narrow_stereo" />
      <param name="verbose" type="bool" value="false"/>
      <param name="scaling" type="double" value="1.00"/>

      <param name="n_planes_max" type="int" value="10"/>
      <param name="point_plane_distance" value="0.02"/>
      
      <param name="n_seeds_per_plane" type="int" value="10"/>
      <param name="cost_pix_in_front" value="0.2"/>
      <param name="cost_pix_unknown" value="0.5"/>

      <param name="show_colorized_planes" type="bool" value="true"/>
      <param name="show_convex_hulls" type="bool" value="false"/>
      <param name="show_lines" type="bool" value="true"/>
      <param name="show_images" type="bool" value="false"/>
      <param name="show_corners" type="bool" value="true"/>
      <param name="show_rectangles" type="bool" value="true"/>
      <param name="save_images" type="bool" value="false"/>

      <param name="select_frontplane" type="int" value="-1"/>
      <param name="max_lines" type="int" value="200"/>
      <param name="max_corners" type="int" value="500"/>

      <param name="min_precision" type="double" value="0.8"/>
      <param name="min_recall" type="double" value="0.8"/>

      <param name="rect_min_size" type="double" value="0.05"/>
      <param name="rect_max_size" type="double" value="1.50"/>
      <param name="rect_max_displace" type="double" value="0.3"/>

    </node>
</launch>

